winui/BusyIndicator/AnimationType.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Content with WinUI BusyIndicator control | Syncfusion
+description: Learn here all about Seven different animation types and features with Syncfusion WinUI BusyIndicator Control.
```

Comment:
description: Learn all about Seven different animation types and features with Syncfusion WinUI BusyIndicator Control here.

Reply:
Changed the content description as mentioned above.